apis/v1alpha2/types.go

```diff
@@ -14,55 +14,32 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package v1alpha2
```

Comment:
nitpick: we have been making `ProviderConfig` / `ProviderConfigUsage` `v1beta1` across other providers.

Reply:
Done. Thanks.